apps/src/code-studio/browserNavigation.js

```diff
@@ -6,11 +6,10 @@
 import {getStore} from '../redux';
 import {setCurrentLevelId} from '@cdo/apps/code-studio/progressRedux';
 
-// Returns whether we can safely navigate between the two given level apps
-// without reloading the whole page.  For now, this only works when moving
-// from a "music" level to another "music" level.
-export function canChangeLevelInPage(currentLevelApp, newLevelApp) {
-  return currentLevelApp === 'music' && newLevelApp === 'music';
+// Returns whether we can safely navigate between the two given levels
+// without reloading the whole page.
+export function canChangeLevelInPage(currentLevel, newLevel) {
+  return currentLevel?.usesLabContainer && newLevel?.usesLabContainer;
```

Comment:
should this be `usesLab2` for consistency?

Reply:
That does make sense.  Related, should `LabContainer` actually be `Lab2Container`?

Reply:
Hmm, actually is there a more specific name we can think of for `LabContainer` that relates to saving/error UI? Asking because the `Lab2` top level file feels more like a "Lab Container"